src/groups/mqb/mqbs/mqbs_filestoreprotocol.h

```diff
@@ -1160,10 +1162,20 @@ struct MessageRecord {
     static const int  k_CAT_START_IDX = 0;
     static const char k_CAT_MASK;
 
+    // PRIVATE CONSTANTS
+    static const int k_REFCOUNT_NUM_LOW_BITS  = RecordHeader::k_FLAGS_NUM_BITS;
+    static const int k_REFCOUNT_NUM_HIGH_BITS = 8;
+
+    static const int k_REFCOUNT_LOW_BITS_MASK = (1u
+                                                 << k_REFCOUNT_NUM_LOW_BITS) -
+                                                1;
```

Comment:
```suggestion
    static const int k_REFCOUNT_LOW_BITS_MASK = RecordHeader::k_FLAGS_MASK;
```
This is more resilient to changes in `mqbs::RecordHeader`.